python/ray/_private/monitor.py

```diff
@@ -28,6 +28,7 @@
 from ray._private.ray_logging import setup_component_logger
 from ray.experimental.internal_kv import _internal_kv_put, \
     _internal_kv_initialized, _internal_kv_get, _internal_kv_del
+from ray._raylet import connect_to_gcs, disconnect_from_gcs
```

Comment:
since we will be removing the ray.* imports from these files, is there a way this can be in ray.autoscaler.*?

Reply:
Or can we actually make this grpc based?

Reply:
Actually connect_to_gcs and disconnect_from_gcs are built on gcs directly.

We don't use grpc based method instead of having a wrapper on cpp gcs is because it can share the same connection if we actually have a worker there. Since we already have the issue that there are too many connections, we probably don't want it here. 

Since internal_kv is not decoupled from ray, it makes sense to import it from ray, right?